web/src/beta/features/AccountAndWorkSpaceSetting/innerPages/AccountSetting/PasswordModal/index.tsx

```diff
@@ -0,0 +1,200 @@
+  }, [updatePassword, handleClose, password, passwordConfirmation]);
+
+  useEffect(() => {
+    if (
+      password !== passwordConfirmation ||
+      (passwordPolicy?.highSecurity &&
+        !passwordPolicy.highSecurity.test(password)) ||
+      passwordPolicy?.tooShort?.test(password) ||
+      passwordPolicy?.tooLong?.test(password)
+    ) {
+      setDisabled(true);
+    } else {
+      setDisabled(false);
+    }
+  }, [password, passwordConfirmation, passwordPolicy]);
+
+  const isMatchPassword =
+    password !== passwordConfirmation && passwordConfirmation;
+
+  return (
+    <Modal size="small" visible={isVisible}>
+      <ModalPanel
+        title={t("Change password")}
+        onCancel={handleClose}
+        actions={[
+          <Button
+            key="Change password"
+            title={t("Change password")}
+            appearance="dangerous"
+            disabled={disabled}
+            onClick={handleSave}
+          />
+        ]}
+      >
+        <ModalContentWrapper>
+          <div>
+            <SubText>
+              <Typography size="body" weight="bold">
+                {t(
+                  `In order to protect your account, make sure your password is unique and strong.`
+                )}
+              </Typography>
+            </SubText>
+            <PasswordField direction="column">
+              <Typography size="body" weight="bold">
+                {t("New password")}
+              </Typography>
+              <TextInput
+                value={password}
+                onChange={handlePasswordChange}
+                type="password"
+              />
+              {password ? (
+                <PasswordMessage size="body" weight="bold">
+                  {regexMessage}
+                </PasswordMessage>
+              ) : undefined}
+            </PasswordField>
+            <PasswordField direction="column">
+              <Typography size="body" weight="bold">
+                {t("New password (for confirmation)")}
+              </Typography>
+              <TextInput
+                value={passwordConfirmation}
+                onChange={setPasswordConfirmation}
+                type="password"
+              />
+              {isMatchPassword ? (
+                <PasswordMessage size="body" weight="regular" color="red">
+                  <span>
+                    <Icon icon="warning" size="large" color="red" />
+                  </span>
+                  {t('"repeatPassword" Passwords need to match')}
+                </PasswordMessage>
+              ) : undefined}
+            </PasswordField>
+          </div>
+        </ModalContentWrapper>
+      </ModalPanel>
+    </Modal>
+  );
+};
+
+const ModalContentWrapper = styled("div")(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  gap: theme.spacing.large,
+  padding: theme.spacing.large,
+  background: theme.bg[1]
+}));
+
+const SubText = styled.div`
+  margin: ${({ theme }) => `${theme.spacing.large}px auto`};
```

Comment:
Please check the layout solution on design. Basically we locate this together with two password fields vertically in a flex box, and making the space between them by gap.

Reply:
BTW please check all text font weight as well, i think they're not in `bold` on design.

Reply:
Already change font-weight to `regular`. And delete useless div, style component to use existed gap.
https://github.com/reearth/reearth-visualizer/pull/1155/commits/7b82f8cad5f29e78af4919388373919819c1ab73